typeshed/__init__.py

```diff
@@ -1,4 +1,53 @@
-import os
+# -*- coding: utf-8 -*-
+"""Typeshed is a collection of type annotations for the standard library and
+third-party libraries.
+
+The type annotations in question are stored in so-called "stub files".  They're
+Python-like files with the *.pyi file extension.  Those files are parseable by
+the built-in Python AST but are not executable.  Functions and methods in stub
+files don't contain bodies.  Variables and attributes don't contain real values.
+
+Typeshed is organized in the following directory structure:
```

Comment:
Nice, but this has some overlap with the description in CONTRIBUTING.md. Maybe more should be in there? Or in README.md? I'm trying to avoid a situation where you have to piece the full details from too many different sources -- often when something changes some of the documentation sources isn't updated and causes confusion.

Reply:
I understand. I'd like to keep this because README.md is not detailed enough and CONTRIBUTING.md isn't part of the distribution. I'd like `help()` to be descriptive enough on its own.